routes/stories.js

```diff
@@ -20,6 +20,8 @@ router.get("/new", requireAuth, csrfProtection, (req, res) => {
   res.render("create-story", {
     csrfToken: req.csrfToken(),
     story: {},
+    username: res.locals.user.username,
+    email: res.locals.user.email
```

Comment:
Anything in `res.locals` can be accessed in pug automatically with `locals.`. These do not need to be passed in.